libopenage/util/compiler.h

```diff
@@ -8,25 +8,55 @@
  *
  * May contain some platform-dependent code.
  */
+ #include <ciso646>
 
 // pxd: from libcpp.string cimport string
 #include <string>
 
+/**
+ * DLL entry-point decorations.
+ */
+#if defined(_WIN32)
+	#if defined(libopenage_EXPORTS)
+		#define LIBOPENAGE_API __declspec(dllexport)
+	#else
+		#define LIBOPENAGE_API __declspec(dllimport)
+	#endif /* libopenage_EXPORTS */
+#else
+	#define LIBOPENAGE_API
+#endif
+
+#if defined(_MSC_VER)
+#ifndef HAVE_SSIZE_T
+typedef int ssize_t;
```

Comment:
Ah, the `ssize_t` is 32 bit wide from python because of your 32 bit platform i guess. So if we define it the way python does, it should be fine?